### PR TITLE
feature: crossmint

### PR DESCRIPTION
**Pull request by Nidh-i-Sharma**

### CODE REVIEW

File: backend/src/controllers/nftController.ts [hunk 1]
```diff
@@ -11,3 +11,47 @@ export const getNFTCount = asyncHandler(async (_req, res) => {
...
+  const reqBody = JSON.stringify({
+    "mainnet": false,
+    "metadata": {
+      "name": req.body.name,
+      "image": req.body.image,
+      "description": req.body.description
+    },
+    "recipient": recipient
+  });
+  const crossmintURL = `https://www.crossmint.io/api/2022-06-09/collections/${collectionName}/nfts` ;
```
Comment (naruto0913) [inline backend/src/controllers/nftController.ts:36]: @Nidh-i-Sharma 
Save `ttps://www.crossmint.io/api/2022-06-09/collections` in .env and use that env info

File: backend/src/controllers/nftController.ts [hunk 2]
```diff
@@ -11,3 +11,47 @@ export const getNFTCount = asyncHandler(async (_req, res) => {
...
+    res.status(400);
+    throw new Error("Incomplete Data");
+  }
+  const reqHeader = {
+    "x-client-secret": process.env.PROJECT_ID,
+    "x-project-id":process.env.CLIENT_SECRET,
+    "Content-Type":'application/json'
+  };
+  const collectionName = 'NFT';
+  const recipient = "poly:0x12618f45ff6e841470bf71f428aae41ee5bc3c39"
```
Comment (naruto0913) [inline backend/src/controllers/nftController.ts]: @Nidh-i-Sharma 
receipt must came from input parameters, not static

File: backend/src/controllers/nftController.ts [hunk 3]
```diff
@@ -40,14 +40,14 @@ export const postNFT = asyncHandler(async  (req, res) => {
...
-      post.save();
-      // res.send(result)
+      // console.log(result)
+      // const post = new NFT({
+      //   // Name: result.name,
+      //   // Image: result.image,
+      //   // Description: result.desciption
+      // })
+      // post.save();
+      res.send(result)
```
Comment (naruto0913) [inline backend/src/controllers/nftController.ts:50]: @Nidh-i-Sharma 
Why commented out all these lines?

File: backend/src/controllers/nftController.ts (hunk 2 above)
Comment (Nidh-i-Sharma) [reply backend/src/controllers/nftController.ts]: yes, I'll do that.

**Pull request merged**